src/Files.Core/Data/Enums/AppThemeMode.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.Core.Data.Enums
+{
+	public enum AppThemeMode
+	{
+		Default,
```

Comment:
Should this be "System"?

Reply:
I don't have preferences. Would like to rename to System?

Reply:
As long as we keep the existing user settings